crates/payout_curve/src/lib.rs

```diff
@@ -30,6 +32,93 @@ pub struct PayoutPoint {
     pub extra_precision: u16,
 }
 
+#[derive(Clone, Copy)]
+pub struct PartyParams {
+    /// How many coins the party is wagering.
+    margin: Amount,
```

Comment:
I admit that the important thing is that the arguments in the constructor are strongly typed. I only kept it because I liked not needing to document the type haha.

I can use `u64` instead, it might be more correct.